.github/workflows/helm-chart-release.yml

```diff
@@ -88,7 +88,8 @@ jobs:
           cat charts/kubewarden-defaults/README.md >> charts/README.md
           echo >> charts/README.md
           cat charts/kubewarden-crds/README.md >> charts/README.md
-          cp -f charts/README.md ./to-gh-pages
+          cp -f charts/README.md ./to-gh-pages/
+          cp -f artifacthub-repo.yml ./to-gh-pages/
```

Comment:
Do we need to do this because artifacthub reads the file from the helm chart repo url?

Reply:
yep, exactly. It's a bit clunky, but I hope it works.